crypto-ffi/Makefile.toml

```diff
@@ -274,7 +244,7 @@ dependencies = ["bindings-kotlin-jvm"]
 condition = { platforms = ["linux"] }
 
 [tasks.jvm-darwin]
-dependencies = ["jvm-aarch64-darwin", "jvm-x86-darwin"]
+dependencies = ["jvm-aarch64-darwin"]
```

Comment:
If this is the only dependency, then it makes sense to rename `jvm-aarch64-darwin` to just `jvm-darwin` and we can drop this one.